test-addon/src/studySetup.js

```diff
@@ -87,8 +86,8 @@ const studySetup = {
   // Optional: testing overrides.
   // Set from prefs in getStudySetup
   testing: {
-    variation: null,
-    firstRunTimestamp: null,
+    variation: "feature-active",
+    expired: true,
```

Comment:
Quick question, final version supports testing variation by picking variation name from pref defined as property inside studySetup e.g.

variationOverridePreference: "cloud.services.shield.variation",

Reply:
Yes.   That should work.